Source/bmalloc/bmalloc/IsoTLSInlines.h

```diff
@@ -35,7 +35,7 @@
 
 #if !BUSE(LIBPAS)
 
-#if BOS(DARWIN)
+#if BENABLE(MALLOC_HEAP_BREAKDOWN)
```

Comment:
Surely this should be consistent with DebugHeap.h. (I think this is the correct way.)

Reply:
fixed